submissions/React-Appwrite-RealTime-Chat.md

```diff
@@ -0,0 +1,43 @@
+# Hackathon Submission: Real-Time Chat App
+
+## GitHub handles of Team Members  
+- @akshayram1
+
+## Project Title  
+**Real-Time Chat App**
+
+## Project Description    
+During the hackathon, we built a **Real-Time Chat Application** using **React JS** and **Appwrite**. The app allows users to register, log in, and chat in real-time. It features an intuitive interface for easy communication, with messages updating instantly across all active users. The backend infrastructure is powered by **Appwrite**, handling authentication, database management, and real-time functionality.
+
+## Inspiration behind the Project  
+The motivation behind this project was to create a fast and reliable communication platform. Real-time communication has become essential in today’s digital world, and we wanted to explore the power of Appwrite to build an efficient chat app. This project was driven by the desire to simplify real-time messaging, making it more accessible and scalable.
+
+## Tech Stack    
+- **React JS**: Used for building the frontend UI.
+- **Appwrite**: A complete backend platform that managed user authentication, real-time messaging, and database operations.
+- **Appwrite Real-Time API**: Implemented for real-time message synchronization.
+- **Appwrite Databases**: For storing user details and chat messages.
+- **Appwrite Authentication**: To manage user sessions, registration, and login functionality.
+
+### Usage of Appwrite
+- **Appwrite Authentication**:  
+  Implemented for handling user sign-ups, logins, and session management securely.
+  
+- **Appwrite Databases**:  
+  Used to create and store chat messages in the `messages` collection, including attributes such as `user_id`, `username`, and `message body`.
+
+- **Appwrite Real-Time API**:  
+  Utilized to ensure that messages are updated in real time across all connected users as soon as a new message is added to the database.
+
+## Project Repo  
+[https://github.com/your-github-handle/real-time-chat-app]

```

Comment:
@akshayram1 please add the correct project repo here to your submission

Reply:
```suggestion
https://github.com/akshayram1/React-Appwrite-RealTime-Chat
```

@akshayram1 is this the correct repo?

Reply:
@akshayram1 I'm adding this for now so that you have a valid eligible submission. If you want to change this, you can create a new PR before the end of the hackathon